compete/rc-fall.html

```diff
@@ -9,101 +9,118 @@
+          This year's mechanical design challenge will likely be a timed challenge. Students will receive a task and materials, and will be given a set amount of time to accomplish the goal. We want to give students the opportunity to tackle a design challenge on the spot, with a level playing field for everyone involved. To keep the challenge a surprise, minimal details will be released until the day of the event.
     	</div>
+          <div class="notecard" style="float:right">
+            <div class="big-txt"> <i class="fa fa-users"></i> 
+            </div>
+            <h3>Speaker Panel</h3>
+            <div>In the hopes of exposing our students to a variety of perspectives and voices, we hold to hold a speaker panel on a topic related to post-secondary school STEM. Check back for more details soon!</div>
+          </div>
     	</div>
   	</div>
     </div>
 
 <div class="tab-pane fade in" id="external">
-<div class="panel panel-default panel-schedule">
+  <h2>Fall Comp is on October 21, 2017. Check back later for details on the 2017 schedule!</h2>
+<!--<div class="panel panel-default panel-schedule">-->
 
-  <div class="panel-heading">Schedule</div>
-  <div class="panel-body">
-<div class="table-responsive">
-<table class="table table-striped table-bordered" style="text-align: center">
-    <colgroup>
-	<col class="col-md-1">
-	<col class="col-md-3">
-	<col class="col-md-3">
-    </colgroup>
-    <thead>
-	<tr style="text-align: center">
-	    <th>Time</th>
-	    <th style="text-align: center">Fall Competition</th>
-	    <th style="text-align: center">MDC</th>
-	</tr>
-    </thead>
-	<tr>
-	    <th scope="row">8:00 AM</th>
-	    <td colspan="2">Check In</td>
-	</tr>
-	<tr>
-	    <th scope="row">9:00 AM</th>
-	    <td colspan="2">Drivers and Captains Meeting</td>
-	</tr>
-	<tr>
-	    <th scope="row">9:30 AM</th>
-	    <td>Practice Matches</td>
-	    <td> </td>
-	</tr>
-	<tr>
-	    <th scope="row">10:00 AM</th>
-	    <td>Prelim Matches</td>
-	    <td>Test Trials and <br>Presentations/Q&amp;A</td>
-	</tr>
-	<tr>
-	    <th scope="row">12:00 PM</th>
-	    <td colspan="2">Lunch</td>
-	</tr>
-	<tr>
-	    <th scope="row">1:00 PM</th>
-	    <td>Prelim Matches</td>
-	    <td>Final Trials</td>
-	</tr>
-	<tr>
-	    <th scope="row">2:00 PM</th>
-	    <td>Alliance Selection</td>
-	    <td>Final Trials</td>
-	</tr>
-	<tr>
-	    <th scope="row">2:30 PM</th>
-	    <td>Elimination Matches</td>
-	    <td>Final Trials</td>
-	</tr>
-	<tr>
-	    <th scope="row">4:00 PM</th>
-	    <td colspan="2">Final Match</td>
-	</tr>
-	<tr>
-	    <th scope="row">4:30 PM</th>
-	    <td colspan="2">Awards Ceremony</td>
-	</tr>
-	<tr>
-	    <th scope="row">5:00 PM</th>
-	    <td colspan="2">Close</td>
-	</tr>
-</table>
-</div>
-</div>
+
+  <!--<div class="panel-heading">Schedule</div>-->
```

Comment:
These comments look painful (and painful to remove). Or did you use an IDE?

Reply:
Ah it's a vim plugin i think lol